engines/dfc_provider/config/routes.rb

```diff
@@ -6,5 +6,6 @@
     resources :catalog_items, only: [:index, :show, :update]
     resources :supplied_products, only: [:create, :show, :update]
   end
+  resources :enterprise_groups, only: [:show]
```

Comment:
I'm probably overthinking it, but.. this looks like it's not a DFC standard endpoint, and I wondered if it could be exposed on the enterprises endpoint?
 We obviously need to differentiate an Enterprise and EnterpriseGroup, but maybe that could be part of the ID segment. Eg `/api/dfc/enterprises/group-123`. Would that make it more DFC-compliant?

Reply:
> Eg /api/dfc/enterprises/group-123. Would that make it more DFC-compliant?

The DFC doesn't specify the names of the URLs, as far as I know. We could merge the endpoints but I think that it would make the code more complicated for no benefit.